build.gradle

```diff
@@ -140,14 +141,15 @@ tasks.withType(PublishToMavenRepository) {
   }
 }
 
-signing {
-  sign publishing.publications.mavenJava
-  if (System.env['CI']) {
-    useInMemoryPgpKeys(System.env.GPG_PRIVATE_KEY, System.env.GPG_PASSPHRASE)
+if (!project.gradle.startParameter.taskNames.contains('publishToMavenLocal')) {
```

Comment:
Don't sign the maven package if publishing to local maven repository (for multi-repository development)